cmd/utils.go

```diff
@@ -831,3 +834,41 @@ func execAndWaitWithWorkDirReturnErr(command string, args []string, logger appso
 	}
 	return err
 }
+
+func checksum256TestFile(newFileName string, oldFileName string) (bool, error) {
+	var checkValue bool
+	fmt.Println("Checksum 256")
```

Comment:
Do these get printed on console when user runs appsody cli? Should these be info logs ?